include/deal.II/numerics/vector_tools.h

```diff
@@ -788,7 +796,8 @@ namespace VectorTools
    * mass matrix.
    * @param[in] function The function to project into the finite element space.
    * @param[out] vec The output vector where the projected function will be
-   * stored in. This vector is required to be already initialized.
+   * stored in. This vector is required to be already initialized and must not
+   * be ghosted.
```

Comment:
-> and must not have any ghost elements.

I think of the term "ghosted" to be colloquial.